.github/workflows/build-push-images.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and push images
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
we probably don't want to update the 'latest' image on every PR that gets to the master branch but rather to update from stable branches, no?

Reply:
For the stable branches, I would suggest doing something like
`stable-2.3` pushes images to `2.3` tag and manually we would create the Z versions.
As for the main branch, I would put it either to the latest or some dev tag like `unstable`. If we will have some larger e2e tests it would be best to use the latest code.

Reply:
> For the stable branches, I would suggest doing something like `stable-2.3` pushes images to `2.3` tag and manually we would create the Z versions.

the more it is automated, the better it is.
what's wrong with the current approach that pushes images from a stable branch `release-v2.3.4` to tag `release-v2.3.4`?

> As for the main branch, I would put it either to the latest or some dev tag like `unstable`. If we will have some larger e2e tests it would be best to use the latest code. And the "latest" would be just the latest release.

looking at [konveyor/forklift-controller on quay.io](https://quay.io/repository/konveyor/forklift-controller?tab=tags) I see it was done like what you wrote above - just the tag is 'main' and the 'latest' is probably updated automatically so that's fine